assets/src/less/global-admin.less

```diff
@@ -276,21 +287,33 @@
 
     @media screen and (min-width: 376px) and (max-width: 576px) {
```

Comment:
Should we care about `(min-width: 376px)` in the above code section? Pls see the recommended screen sizes
- https://getbootstrap.com/docs/5.0/layout/breakpoints/
- https://tailwindcss.com/docs/screens

Reply:
Bhai,
@media screen was impliment prev PR . 
I think we can skip this refactor as it is working